src/controllers/sessionLockController.ts

```diff
@@ -245,6 +247,7 @@ export default class SessionLockController {
             pipe(
               AP.sequenceT(TE.ApplicativeSeq)(
                 ...this.buildInvalidateUserSessionTask(fiscalCode),
+                this.clearInstallation(fiscalCode),
```

Comment:
If this operation fail, we cannot lock the user session.

Reply:
Added a comment to clarify the reasoning behind the chosen order. Thanks. [569ee4c](https://github.com/pagopa/io-backend/pull/1080/commits/569ee4c8084032a3cfe6388fdf85f719adb6f788)